### DISCUSSION TITLE
Really needs Optimization: datapack(Commands and Tags)

### DISCUSSION THREAD

**Discussion by davidromrell** [FabricMC/fabric-api#3703] [Player support] [answered]
Thanks
I noticed that when I made any commands or tags I got down to 0FPS consistently regardless of what they were (although its worst with ticking in tags.)

**Comment by haykam821** [comment, marked as answer]
Fabric doesn't typically implement performance changes; you should look to third-party mods for that. Your request is a bit vague as well. If you think something can be optimized, you should provide specific examples of the slowdowns and how the algorithm can be optimized. Right now, you're pointing to a general topic as a problem even though most people can use commands and tags without greatly affecting their game's performance.
